### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by TahenyBELGUITH**
In this part
I add accessibility where:
- Page titles: no issues found
- Image text alternatives: no issues found
- Text headings: no issues found
- Color contrast: no issues found
- Resize: no issues found
- Interaction: no issues were found
- Moving content: no issues found
- Multimedia: no issues found
- The basic structure of the page: no issues found

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,17 @@
 > -I create a desktop version app
 > -I integrate Figma UI by using Html CSS
 
+- I add google analytics
+- I add accessibily
```
Comment (rloterh) [inline README.md:8]: - [ ] There are a number of accessibility points which were not highlighted in your report: Kindly ensure to make assessment on the following points and include reports on them accordingly as well:
- Page titles
- Image text alternatives `** (alt text)`
- Text headings
- Color contrast
- Resize
- Interaction
- Moving content
- Multimedia
- The basic structure of the page

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Tahtouha, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- No linter errors

## Required Changes ♻️
- [ ] _Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -5,40 +5,43 @@
...
+        gtag('config', 'G-QTG297H0DH');
+    </script>
 </head>
 
 <body>
     <header>
-
-        <nav class="toolbar">
+        <nav class="toolbar sticky">
             <a href="./index.html"><img class="toolbar-img2" src="images/logo.svg" alt="Logo"></a>
```
Comment (hemant-soni-vst-au4) [inline index.html:29]: - [ ] As highlighted by a previous reviewer the sub-contents of the navbar are well spaced from the original design. Right now there is a need for a little more margin or padding. Kindly fix this issue.

yours
image

Original 
image

File: README.md [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
+  - add accessibily:
+  - Page titles
+  - Image text alternatives
+  - Text headings
+  - Color contrast
+  - Resize
+  - Interaction
+  - Moving content
+  - Multimedia
+  - The basic structure of the page
```
Comment (hemant-soni-vst-au4) [inline README.md:16]: - [ ] As per project requirements and issues highlighted by the previous reviewer, We have to prepare a report listing all points that you have checked, and the result of your evaluation. Add our report to the description of your pull request.

image

**Review by hemant-soni-vst-au4 — CHANGES_REQUESTED** [review]
Hi @TahenyBELGUITH , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text
 ## Required Changes ♻️
_Check the comments under the review._

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shaher-11 — APPROVED** [review]
#### Hello @TahenyBELGUITH,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**